### ISSUE TITLE
xcode9.2 error

### ISSUE THREAD

**Issue by anyinst** [obfuscator-llvm/obfuscator#100]
fatal error: 'mutex' file not found
#include <mutex> 
         ^~~~~~~
1 error generated.

**Comment by longv2go** [comment]
@anyinst Which branch of obfuscator do you build? I build llvm-4.0 and copy clang to xcode9.2, it come out error 'error: cannot specify -o when generating multiple output files'.

**Comment by csc-EricWu** [comment]
@longv2go me too

**Comment by a130785** [comment]
@longv2go me too

**Comment by exorxw** [comment]
i think you need to upgrade the llvm 5.0.1

**Comment by Naville** [comment]
Nope. Build libcxx and libcxxabi. For example if the branch is called 4.0, then download version 4.0 of libcxx and libcxxabi from https://releases.llvm.org , put into projects/ then restart building.(Might need to build again from scratch for cmake to recognize the new files)
